src/frontend/org/voltdb/planner/PlanAssembler.java

```diff
@@ -113,6 +115,15 @@ public ParsedResultAccumulator(boolean orderIsDeterministic,
         }
     }
 
+    private static final Set<ExpressionType> PERCENTILE_EXPRESSIONS = Collections.synchronizedSet(EnumSet.of(
```

Comment:
(i'm probably missing something here).. why does this need to be a `synchronizedSet`?  Could `unmodifiableSet` work instead? i.e. doesn't seem like this is written to again, so you could get a similar effect with less overhead..

Reply:
Good call. That's me blindly following the recommendations in the EnumSet documentation without considering how my use case differs.

Switched to `unmodifiableSet`.